tests/test_manifest.py

```diff
@@ -86,6 +90,101 @@ def test_manifest_with_anchor(tmpdir, monkeypatch):
     assert manifest.enable_build_targets('bar') == []
 
 
+def test_manifest_with_anchor_and_postfix(tmpdir, monkeypatch):
+    yaml_file = tmpdir / 'test.yml'
+    yaml_file.write_text(
+        """
+.base_depends_components: &base-depends-components
+  depends_components:
+    - esp_hw_support
+    - esp_rom
+    - esp_wifi
+
+examples/wifi/coexist:
+  <<: *base-depends-components
+  depends_components+:
+    - esp_coex
+  depends_components-:
+    - esp_rom
+""",
+        encoding='utf8',
+    )
+
+    manifest = Manifest.from_file(yaml_file)
+    assert manifest.depends_components('examples/wifi/coexist') == ['esp_coex', 'esp_hw_support', 'esp_wifi']
+
+    yaml_file.write_text(
+        """
+.base: &base
+  enable:
+    - if: IDF_VERSION == "5.2.0"
+    - if: IDF_VERSION == "5.3.0"
+
+foo:
+  <<: *base
+  enable+:
+    - if: IDF_VERSION == "5.2.0"
+      temp: true
+    - if: IDF_VERSION == "5.4.0"
+      reason: bar
+""",
+        encoding='utf8',
+    )
+    s_manifest_dict = parse(yaml_file)
+
+    yaml_file.write_text(
+        """
+foo:
+    enable:
+        -   if: IDF_VERSION == "5.2.0"
```

Comment:
I know the original task says we shouldn't override it, but if we don't provide one way to override the dict, there's no way to achieve it. Maybe we should override it with the same `if`.

WDYT?

Reply:
Let's do override!